lms/djangoapps/learner_home/mock_views.py

```diff
@@ -7,21 +7,23 @@
 import json
 from os import path
 
-from django.contrib.auth.decorators import login_required
-from django.views.decorators.http import require_GET
-from common.djangoapps.util.json_request import JsonResponse
+from rest_framework.response import Response
+from rest_framework.generics import RetrieveAPIView
+
+import logging
```

Comment:
Probably don't need the logging anymore.  That was for debug, and the calls have been removed.  V low-priority tho.